Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -1,20 +1,16 @@
 (function initTargetBlank() {
-    function targetBlank() {
+    function addTargetBlank() {
         const links = document.querySelectorAll('a');
         const currentHostname = window.location.hostname;
 
         for (let i = 0; i < links.length; i++) {
-            if (!links[i].href.match(/^mailto:/) &&
-                (links[i].hostname !== currentHostname &&
-                (!links[i].href.match(/^javascript:/i)))) {
+            if (links[i].hostname !== currentHostname &&
+                // eslint-disable-next-line no-script-url
```

Comment:
Please follow refer to the last paragraph of https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Docs/JavaScript.md#eslint-rules.

Add the comment above the existing one.